tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -191,7 +191,7 @@ def run_test(self, setup_info):
         test_ports_info = setup_info['test_ports']
         queues = [self.storm_handle.pfc_queue_idx]
 
-        with send_background_traffic(self.dut, self.ptf, queues, selected_test_ports, test_ports_info):
+        with send_background_traffic(self.dut, self.ptf, queues, selected_test_ports, test_ports_info, pkt_count=500):
```

Comment:
Seems the function doesn't take the parameter `pkt_count`?
https://github.com/sonic-net/sonic-mgmt/blob/0028a82d18b410cae141170a625999efa8548ee7/tests/common/helpers/pfcwd_helper.py#L492

Reply:
This PR should based on the PR https://github.com/sonic-net/sonic-mgmt/pull/15870